### ISSUE TITLE
How to use `Config/Recaptcha` on release since `v2.0.0`

### ISSUE THREAD

**Issue by ajimsofwan** [PHPDevsr/reCaptcha-Codeigniter4#10]
Argument 1 passed to PHPDevsr\Recaptcha\Recaptcha::__construct() must be an instance of PHPDevsr\Recaptcha\Config\Recaptcha or null

Can't use
use Config\Recaptcha as RecaptchaConfig;
For now i'm using this
use PHPDevsr\Recaptcha\Config\Recaptcha as RecaptchaConfig;

**Comment by ddevsr** [comment]
@ajimsofwan I already push to repo to fix this, before release on `v2.0.0`

try
```bash
composer require phpdevsr/recaptcha-codeigniter4:dev-dev
```

Now can use helper for `getWidget()`, `getScriptTag()` and `verifyResponse()`

```php
<?php

namespace App\Controllers;

class Home extends BaseController
{
    public function index(): string
    {
        helper('recaptcha');

        $data = [
            'scriptTag' => getScriptTag(),
            'widgetTag' => getWidget(),
        ];

        $captcha = $this->request->getPost('g-recaptcha-response');
        $response = verifyResponse($captcha);

        if (isset($response['success']) and $response['success'] === true) {
            echo "You got it!";
        }

        return view('welcome_message', $data);
    }
}
```

![image](https://github.com/PHPDevsr/reCaptcha-Codeigniter4/assets/97607754/8c32773a-9f67-4c9c-a858-61ca82b6825d)
I have tested using helper

**Comment by ddevsr** [comment]
Or you can use `service('recaptcha')`

```php
$recaptcha = service('recaptcha');

$recaptcha->getScriptTag(); // For script tag
$recaptcha->getWidget(); // For widget
$recaptcha->verifyResponse($response); // For verify
```
